### PR TITLE
[sd-3666]-UI: Newly created desk doesn't reflect the stage count

### PR DESCRIPTION
**Pull request by sjunaid**
Fix covers SD-3676: Update is not initiated when adding/removing a stage to/from a desk

### CODE REVIEW

File: client/spec/helpers/desks.js [hunk 1]
```diff
@@ -51,6 +51,19 @@ function Desks() {
     };
 
     /**
+     * Returns the stage count for named desk on desks settings list
+     * @param {string} name of desk
+     * @return {string} stage count
```
Comment (plamut) [inline client/spec/helpers/desks.js]: This doesn't seem right, since the method actually returns a _promise_ which is resolved with the stage count?

**Comment by akintolga** [comment]
:+1:

**Comment by ioanpocol** [comment]
PR https://github.com/superdesk/superdesk/pull/1408 has fixed the e2e tests problems.
Please rebase.

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**